ide/git/src/org/netbeans/modules/git/options/GitOptionsPanel.java

```diff
@@ -149,7 +157,7 @@ private void initComponents() {
                     .addComponent(jLabel2)
                     .addComponent(txtProjectAnnotation, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
                     .addComponent(btnAddVariable))
-                .addContainerGap(javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
+                .addContainerGap(10, Short.MAX_VALUE))
```

Comment:
could you go though all check boxes in this panel and do the following:
 - right click -> Edit Layout Space...
 - make sure Top and Bottom is set to "default medium"

so that we don't have fixed gaps defined, this would look inconsistent if if some UI settings change.

Reply:
Fixed.